src/frontend/overlay/src/components/atoms/ContestLabels.jsx

```diff
@@ -115,15 +117,36 @@ const TaskResultLabelWrapper2 = styled.div`
   color: #fff;
 `;
 
-// TODO: fts start
-const ICPCTaskResultLabel2 = ({ problemResult: r, ...props }) => {
+
+const StarIconWrap = styled.div`
+    position: absolute;
+    width: 33px;
+    height: 33px;
```

Comment:
Do I need to make this value constant? (maybe STAR_SIZE in config.js?)

Reply:
Yes, please extract to config.js

Reply:
fixed